WalletWasabi.Documentation/ClientRelease/ClientDeployment.md

```diff
@@ -1,68 +1,58 @@
-# 1. Final tests
+# Final tests
 
-1. Go to https://github.com/zkSNACKs/WalletWasabi/releases.
+1. Draft a new release.
 2. Check the exact **date** of the last release and the **name** of the last PR.
 3. List the PR-s in order, open the [link and (adjust date!)](https://github.com/zkSNACKs/WalletWasabi/pulls?q=is%3Apr+merged%3A%3E%3D2019-07-07+sort%3Aupdated-asc).
 4. Go trough all PR, create the Release Notes document and the Final Test issue. Create test cases according to PR-s and write a list. [Release Notes format](https://github.com/zkSNACKs/WalletWasabi/releases/tag/v1.1.6) and [Final Test format](https://github.com/zkSNACKs/WalletWasabi/issues/2227).
 5. Go trough all issues and pick the [important ones (adjust date!)](https://github.com/zkSNACKs/WalletWasabi/issues?utf8=%E2%9C%93&q=is%3Aissue+closed%3A%3E%3D2019-07-07+sort%3Aupdated-asc+) and add to Release Notes or to Final Tests if required.
 6. Check Tor status. Never release during a Tor network disruption: https://status.torproject.org/
 7. At the end there will be a Final Test document and a Release Notes document.
 
-# 2. Release candidate
+# Release candidate
 
-1. Go to https://github.com/molnard/WalletWasabi/releases and press Draft a new release.
+1. Go to your own fork of Wasabi and press Draft a new release. Release candidates are not published in the main repository!
```

Comment:
Why not publish RCs on the main repository?
Many if not most open source projects publish them on the main repo, for example bitcoin core has the RCs only under the tags subpage: https://github.com/bitcoin/bitcoin/tags.

Why would anyone trust someone else's personal fork, and how would contributors even know about the existence of these RCs on personal forks?

Reply:
I need to document how we do it now. There is no room for changes at this PR.